src/main/java/impl/ReaderFromFileImpl.java

```diff
@@ -0,0 +1,18 @@
+package impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import service.ReaderFromFile;
+
+public class ReaderFromFileImpl implements ReaderFromFile {
```

Comment:
The class implementation name `ReaderFromFileImpl` does not fully conform to the Java naming conventions. The name could be more descriptive about the type of files it reads, for example, `CsvReaderServiceImpl` if it's intended to read CSV files.